### ISSUE TITLE
Gas estimation failed without revert reason or error message.

### ISSUE THREAD

**Issue by EVMlord** [ethereum/remix-project#4241]
![image](https://github.com/ethereum/remix-project/assets/85406816/6e8fa2c6-cc8a-4081-b6de-7b786bc7c306)

**Comment by MartyMari** [comment]
Hello @EVMlord, have you been able to resolve this issue?

**Comment by LianaHus** [comment]
Which network is this?

**Comment by calmacfadden** [comment]
I reported this months ago on twitter. I might just have to go and fix it myself! 
It happens on all networks.

**Comment by calmacfadden** [comment]
Ok so i decided to look into it. Its being caused by the web3j library. 
The text being shown "Error happened while trying to execute a function inside a smart contract" is hardcoded instead of actually showing the error reported by the contract. 

Its also reported as an issue in the web3j github https://github.com/web3/web3.js/issues/6388

**Comment by masonreid1729** [comment]
Yes, I also received funds through this faucet, but unfortunately, I didn't find any more active wallets to claim according to their rules.

<img width="369" alt="Image" src="https://github.com/user-attachments/assets/b8dea568-d29e-4d6e-a68f-eb1dd5cffa6b" />

<img width="373" alt="Image" src="https://github.com/user-attachments/assets/6edb3f06-24b7-479f-8171-b11afa84e97f" />

**Comment by Aniket-Engg** [comment]
This should be working fine. Feel free to reopen if issue persists
